test/0.8.9/withdrawal-queue.test.js

```diff
@@ -156,4 +156,50 @@ contract('WithdrawalQueue', ([recipient, stranger, daoAgent, user]) => {
       assertBn(await ethers.provider.getBalance(recipient), balanceBefore.add(bn(ETH(50))))
     })
   })
+
+  context('Transfer request', async () => {
```

Comment:
I would add some additional cases:
- auth error
- already claimed request (can't transfer)
- passing the same recipient
- wrong request id